testdata/lint/lint.go

```diff
@@ -62,7 +63,10 @@ func Lint(opts *Options) func() {
 
 			Ω.Expect(subject.Glob(ctx, "*")).To(whenDrained(Ω.BeEmpty()))
 			cancel()
-			Ω.Expect(blank.Commit()).To(Ω.Equal(context.Canceled))
```

Comment:
I think, it was Expect to Equal maybe before errors.Is/As was introduced/widely-used?

I'm switching to errors.Is, but maybe I had to keep unwrapping context.Canceled for older caller code? I'd prefer not to.

Reply:
I think you can do MatchError